third_party/terraform/utils/error_retry_predicates.go

```diff
@@ -25,11 +30,34 @@ var defaultErrorRetryPredicates = []RetryErrorPredicateFunc{
 
 /** END GLOBAL ERROR RETRY PREDICATES HERE **/
 
-func isUrlTimeoutError(err error) (bool, string) {
-	if urlerr, ok := err.(*url.Error); ok && urlerr.Timeout() {
-		log.Printf("[DEBUG] Dismissed an error as retryable based on googleapis.com target: %s", err)
+type temporary interface {
+	Temporary() bool
```

Comment:
Does anything implement this right now?

Reply:
yup - url.Error and net.OpError

Reply:
Thanks! Mind commenting and linking to an example of an implementer?

Also, I think we actually want to call `Temporary()`, right? Since they'll return `false` for non-temporary errors based on https://golang.org/src/net/net.go?s=13212:13333#L384

Reply:
i realized right after and pushed an update ha